third-party/linshare/src/main/java/org/apache/james/linshare/client/LinshareAPI.java

```diff
@@ -35,9 +34,8 @@
 import feign.Feign;
 import feign.Logger;
 import feign.Param;
-import feign.RequestInterceptor;
```

Comment:
I bet `List<ShareResult> share(ShareRequest request)` is no longer required,

as well as `Document uploadDocument(File file)`

Can we try to cleanup a bit this class?

Reply:
Yes, i will